e2e-tests/insurance/cypress/e2e/journeys/your-buyer/change-your-answers/change-your-answers-change-traded-with-buyer-no-to-yes-outstanding-yes.spec.js

```diff
@@ -0,0 +1,98 @@
+import { YOUR_BUYER as FIELD_IDS } from '../../../../../../constants/field-ids/insurance/your-buyer';
+import { INSURANCE_ROUTES } from '../../../../../../constants/routes/insurance';
+import { INSURANCE_FIELD_IDS } from '../../../../../../constants/field-ids/insurance';
+import { summaryList } from '../../../../../../pages/shared';
+import checkSummaryList from '../../../../../../commands/insurance/check-your-buyer-summary-list';
+
+const {
+  TRADED_WITH_BUYER,
+  OUTSTANDING_PAYMENTS,
+  HAS_PREVIOUS_CREDIT_INSURANCE_COVER_WITH_BUYER,
+  PREVIOUS_CREDIT_INSURANCE_COVER_WITH_BUYER,
+  FAILED_PAYMENTS,
+  TOTAL_OUTSTANDING_PAYMENTS,
+  TOTAL_AMOUNT_OVERDUE,
+} = FIELD_IDS;
+
+const { CURRENCY_CODE } = INSURANCE_FIELD_IDS.CURRENCY;
+
+const {
+  ROOT,
+  YOUR_BUYER: { TRADING_HISTORY_CHANGE, CHECK_YOUR_ANSWERS },
+} = INSURANCE_ROUTES;
+
+const fieldId = TRADED_WITH_BUYER;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context(
+  `Insurance - Your buyer - Change your answers - ${TRADED_WITH_BUYER} - No to yes - ${OUTSTANDING_PAYMENTS} as yes - As an exporter, I want to change my answers to the trading history section`,
```

Comment:
Can remove "as yes"

Reply:
Bit confused about the E2E descriptions/file names:

```ts
change-your-answers-change-the-field-no-to-yes-outstanding-yes
```

It other tests we just have e.g

```ts
change-your-answers-change-the-field-no-to-yes
```

Is there different logic for this feature where:
- if "traded with buyer" is changed from no to yes
- **and** "outstanding" is yes
- Do something different?